e-commerce/shop-backend/package.json

```diff
@@ -9,17 +9,20 @@
     "lint-fix": "eslint --fix src"
   },
   "dependencies": {
-    "@dbos-inc/communicator-bcrypt": "^0.8.62",
-    "@dbos-inc/dbos-sdk": "^0.8.62",
+    "@dbos-inc/communicator-bcrypt": "^0.9.25",
+    "@dbos-inc/dbos-sdk": "^0.9.25",
```

Comment:
Shouldn't we use preview here too? (To be consistent with the payment backend?)

Reply:
It won't be long before the preview is published and we can then do both to latest.